Scripts/niceListAlgorithm.py

```diff
@@ -113,4 +113,16 @@ def santaAssign(emails: List, not_allowed: List) -> List:
 
 #print(emailList)
 #print(constraintList)
-print(santaAssign(permuteList(emailList),constraintList))
\ No newline at end of file
+santaWeb = santaAssign(permuteList(emailList),constraintList)
+#print(santaWeb)
+
+#put assignments into database
+try:
+    #open connection to database
+    with sqlite3.connect(pathToDB) as connection:
+        dbCursor = connection.cursor()
+        for ID in emailList:
+            dbCursor.execute('SELECT email FROM people where person_ID = ?', (santaWeb[ID],))

```

Comment:
you can remove the above line I think

Reply:
I did, and it didn't cause issues. I don't understand why though. I thought the connection was closed after the end of the previous with statement.

Reply:
commenting out line 123 should definitely make it fail lol. Did you forget to restart the web server after making changes to the file? Also I was talking about line 125 where you run a select on the db but don't do anything with the results.

Reply:
Ah. Not sure why that was there. I removed line 125.